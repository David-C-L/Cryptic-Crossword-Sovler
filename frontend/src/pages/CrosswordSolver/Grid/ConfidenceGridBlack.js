import './ConfidenceGridBlack.css'
import './ConfidenceGridCell.css'

function ConfidenceGridBlack(props) {
    return (
        <th scope="col" className="grid-column cell grid-cell black-cell" key={props.col} />
    )
}

export default ConfidenceGridBlack;