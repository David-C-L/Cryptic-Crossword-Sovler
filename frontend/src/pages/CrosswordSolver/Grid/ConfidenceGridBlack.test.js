import React from "react";
import { unmountComponentAtNode } from "react-dom";
import renderer from 'react-test-renderer';
import { BrowserRouter as Router } from "react-router-dom";
import ConfidenceGridBlack from "./ConfidenceGridBlack";


let container = null;
beforeEach(() => {
  // setup a DOM element as a render target
  container = document.createElement("div");
  document.body.appendChild(container);
});

afterEach(() => {
  // cleanup on exiting
  unmountComponentAtNode(container);
  container.remove();
  container = null;
});

it('renders correctly', () => {
  const tree = renderer
    .create(<Router><ConfidenceGridBlack /></Router>)
    .toJSON();
  expect(tree).toMatchSnapshot();
});