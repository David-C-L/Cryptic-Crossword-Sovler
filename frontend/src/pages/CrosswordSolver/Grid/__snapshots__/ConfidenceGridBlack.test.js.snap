// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<th
  className="grid-column cell grid-cell black-cell"
  scope="col"
/>
`;
